src/material.cpp

```diff
@@ -51,8 +51,10 @@ mat_burn_data load_mat_burn_data( JsonObject &jsobj )
     return bd;
 }
 
-void material_type::load( JsonObject &jsobj, const std::string & )
+void material_type::load( JsonObject &jsobj, const std::string &src )
 {
+    bool strict = src == "core";
```

Comment:
This check doesn't work anymore. This has to be:
```c++
bool strict = src == "dda";
```
BTW: This applies to all similar checks. We need some kind of a constant `core_mod`.